docs/src/pages/careers/react-engineer-x-grid.md

```diff
@@ -1,130 +0,0 @@
-# React Engineer - xGrid
```

Comment:
I have kept all the markdown of previous roles. It makes it simpler to find back when we want to open similar roles. I wouldn't be against keeping it.

Reply:
Reverted